aten/src/ATen/native/cuda/UnaryOpsKernel.cu

```diff
@@ -1,14 +1,36 @@
+#include <limits>
 #include <ATen/native/UnaryOps.h>
 #include <ATen/native/cuda/Loops.cuh>
 #include <ATen/Context.h>
 #include <ATen/Dispatch.h>
 #include <ATen/native/cuda/Loops.cuh>
 #include <ATen/native/DispatchStub.h>
 #include <ATen/native/TensorIterator.h>
-#include <limits>
 
 namespace at { namespace native {
 
+void bitwise_not_kernel_cuda_impl(TensorIterator& iter) {
+  if (iter.dtype() == ScalarType::Bool) {
+    gpu_kernel(iter, []GPU_LAMBDA(bool a) {
+      return !a;
+    });
+  } else {
+    AT_DISPATCH_INTEGRAL_TYPES(iter.dtype(), "bitwise_cuda", [&]() {
+      gpu_kernel(iter, []GPU_LAMBDA(scalar_t a) -> scalar_t {
+        return ~a;
+      });
+    });
+  }
+}
+
+static void bitwise_not_kernel_cuda(TensorIterator& iter) {
```

Comment:
You can get rid of the static wrapper. It's not really doing anything here.

Previously, when we supported CUDA 8, this pattern was necessary. But now we only have the Windows limitations (i.e. bitwise_not_kernel_cuda_impl can't be static).